src/configs/axios.js

```diff
@@ -3,12 +3,11 @@ import axios from 'axios';
 const baseURL = 'https://authors-heaven.herokuapp.com/';
 const Axios = {};
 if (process.env.NODE_ENV !== 'test') {
-  Axios.instance = axios.create({
-    baseURL,
-    responseType: 'json'
-  });
+  Axios.instance = axios.create({ baseURL });
 } else {
   Axios.instance = axios;
 }
 
+export const FrontendUrl = 'localhost:300';
```

Comment:
@nkpremices localhost:300 or localhost:3000?

Reply:
I'll change that @MemunaHaruna